src/components/TimeSlicing.vue

```diff
@@ -112,19 +112,49 @@ export default defineComponent({
       }
     }
 
-    function exportNetwork() {
+    function exportEdges() {
       if (network.value === null) {
         return;
       }
+      // Slice network in case 'Generate Slices' button
+      // not clicked or updated
+      sliceNetwork();
+      const { slicedNetwork } = store.state;
+
+      // Generate edge table data
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const edges: any[] = [];
+      slicedNetwork.forEach((slice) => {
+        const timeObj = { slice: slice.slice, timeStart: slice.time[0], timeFinish: slice.time[1] };
+        slice.network.edges.forEach((edge) => {
+          const rowObj = Object.assign(edge, timeObj);
```

Comment:
Optional suggestion (that might not even work, lol, it would need some testing):
```suggestion
          const rowObj = {...edge, ...timeObj};
```
I prefer this syntax, but take it or leave it

Reply:
Cool, is there a reason why you prefer that?

Reply:
I just think it looks cleaner. It's a bit more obvious to me that you're taking 2 objects and spreading them out into a new one. `Object.assign` doesn't have that intuitive feel

Reply:
Jack's suggestion is good. FWIW, the other option would be

```javascript
const rowObj = Object.assign({}, edge, timeObj);
```

Making the first argument to `Object.assign()` an empty object prevents mutation of that argument, which is generally something to avoid. Jack's suggestion implicitly avoids this by constructing a new object from scratch.

Reply:
Thank you both for your comments and I will keep that in mind for the future!